helm/coredns-app/templates/configmap.yaml

```diff
@@ -46,6 +42,70 @@ data:
         {{- end }}
     }
 
+    {{- $domains := split " " .Values.cluster.kubernetes.clusterDomain }}
+    {{- $dnsPort := .Values.ports.dns.targetPort }}
+    {{- $clusterIPRange := .Values.cluster.kubernetes.API.clusterIPRange }}
+    {{- $calicoCIDR:= .Values.cluster.calico.CIDR }}
+    {{- $loadbalancePolicy := .Values.loadbalancePolicy }}
+    {{- $prometheus := .Values.ports.prometheus }}
+    {{- $customLog := .Values.configmap.log }}
+    {{- $additionalLocalZones := .Values.additionalLocalZones -}}
+    {{- range $index, $domain := $domains }}
+    {{- if not (has $domain $additionalLocalZones) }}
+    {{ $domain }}:{{ $dnsPort }} {
```

Comment:
we need to add `{{$clusterIPRange}}:{{ $dnsPort }}` and `{{ $calicoCIDR }}:{{$dnsPort}}` here, otherwise this breaks PTRs

Reply:
I changed it, so now we have

```bash
helm template helm/coredns-app/ --set cluster.kubernetes.clusterDomain="my.zone.com cluster.local" --set "additionalLocalZones={my.zone.com}"
```

```yaml
data:
  Corefile: |
    .:1053 {
        cache 30
        errors
        template ANY AAAA {
          rcode NOERROR
        }
        health {
          lameduck 5s
        }
        ready
        log . {
          class denial
          class error
        }
        loadbalance round_robin
        prometheus :9153
        forward . /etc/resolv.conf
        reload
    }
    cluster.local:1053 172.31.0.0/16:1053 192.168.0.0/16:1053 {
        errors
        template ANY AAAA {
          rcode NOERROR
        }
        health {
          lameduck 5s
        }
        ready
        kubernetes cluster.local 172.31.0.0/16 192.168.0.0/16 {
          fallthrough in-addr.arpa ip6.arpa
          pods verified
        }
        log . {
          class denial
          class error
        }
        loadbalance round_robin
        prometheus :9153
        reload
    }
    my.zone.com:1053 {
        errors
        template ANY AAAA {
          rcode NOERROR
        }
        health {
          lameduck 5s
        }
        ready
        kubernetes my.zone.com {
          fallthrough in-addr.arpa ip6.arpa
          pods verified
        }
        log . {
          class denial
          class error
        }
        loadbalance round_robin
        prometheus :9153
        reload
    }
```